### PR TITLE
Feat/help new

### LINKED ISSUE #49: Update help
Some paragraphs in the Help section are outdated.

### PR DESCRIPTION
**Pull request by olastor**
Please look for language mistakes as well

Closes #49

### CODE REVIEW

File: src/app/help/help.html [hunk 1]
```diff
@@ -2,61 +2,87 @@
...
+    </p>
+    <a href="https://github.com/Sirimangalo">
+      <img src="assets/img/GitHub_Logo.png" width="200">
+    </a>
+    <ul class="github-link-rep">
+      <li><a href="https://github.com/Sirimangalo/meditation-plus-angular" target="_blank">Client Repository</a> / <a href="https://github.com/Sirimangalo/meditation-plus-angular/issues" target="_blank">ISSUES</a></li>
+      <li class="github-link-rep"><a href="https://github.com/Sirimangalo/meditation-plus-server" target="_blank">Server Repository</a> / <a href="https://github.com/Sirimangalo/meditation-plus-server/issues" target="_blank">ISSUES</a></li>
+    </ul>
+    <p>We appreciate your support.</p>
+    <h4>Information for smarphone users</h4>
```
Comment (binarious) [inline src/app/help/help.html]: `smartphone`

File: src/app/help/help.html [hunk 2]
```diff
@@ -2,61 +2,87 @@
...
+    <p>If you want to use this app on your smartphone, you can use the <i>Add to Homescreen</i> feature in your mobile browser (Chrome on Android, Safari on IOS) to create an app like experience.</p>
+    <p><b>Please note</b> that meditation alarm is currently not working on smartphones. Therefore you should use a native timer app on your phone until we've found a stable solution for this problem.</p>
```
Comment (binarious) [inline src/app/help/help.html]: that **the** meditation alarm

File: src/app/help/help.html [hunk 3]
```diff
@@ -2,61 +2,87 @@
...
+  </div>
+</md-card>
+<md-card class="help-card">
+  <md-divider></md-divider>
+    <h3 (click)="showSection('Meditation')">Meditation</h3>
+    <div class="content" [hidden]="!sections['Meditation']">
+      <p>The <i>Meditation</i> tab allows you to submit a meditation session to the server and view sessions currently in progress. The list of sessions also shows recently completed sessions below active ones. Above the list there is a graph with the numbers 0-23 below it. The numbers represent hours of the day, and the current hour by UTC standard time is highlighted in yellow. The green bar graphs represent total minutes of meditation per hour (UTC) for the last month, meant to give an idea of when you are most likely to find other meditators online.</p>
```
Comment (binarious) [inline src/app/help/help.html]: Please correct the colors. They are not green and yellow anymore.

File: src/app/help/help.html [hunk 4]
```diff
@@ -2,61 +2,87 @@
...
+      <p>The <i>Commit</i> tab allows you to sign up for various commitments.</p>
+    </div>
+  <md-divider></md-divider>
+    <h3 (click)="showSection('Profile')">Profile</h3>
+    <div class="content" [hidden]="!sections['Profile']">
+      <p>Clicking on a users avatar will load their <i>Profile</i> page.</p>
+      <h4>Settings</h4>
+      <ul>
+        <li>You can select the information displayed for your profile by clicking on <button md-mini-fab><md-icon>edit</md-icon></button>.</li>
+        <li>In your profile settings you can choose a default sound for your meditations and change your username, password, description and country.</li>
```
Comment (binarious) [inline src/app/help/help.html]: Wen don't have a username anymore.

File: src/app/help/help.component.ts [hunk 5]
```diff
@@ -13,6 +13,17 @@ export class HelpComponent {
 
   version: string = 'dev';
 
+  sections = {
```
Comment (binarious) [inline src/app/help/help.component.ts]: Why do we need this?

Comment (olastor) [inline src/app/help/help.component.ts]: Do you mean the layout in general?

Comment (binarious) [inline src/app/help/help.component.ts]: No this `sections` variable. The toggling should be possible with `currentSection` only.

File: src/app/help/help.component.ts [hunk 6]
```diff
@@ -24,4 +26,12 @@ export class HelpComponent {
         this.version = res.version;
       });
   }
+
+  showSection(whichSection) {
+    if (whichSection === this.currentSection){
```
Comment (binarious) [inline src/app/help/help.component.ts]: `this.currentSection = whichSection === this.currentSection ? whichSection : ''`?

**Comment by binarious** [comment]
@yuttadhammo @bkeevil Can someone of you review the text changes?

**Comment by olastor** [comment]
Readable version for review [here](https://jsfiddle.net/kwxy426n/)

**Comment by bkeevil** [comment]
I can do some more work on this page if @olastor is done with his edits?

**Comment by olastor** [comment]
@bkeevil If you mean the help page, it's already in 'master'. So yes, you can change and create a pull request whenever you like.

**Pull request merged**